tasks-by-users/anask601/Responsive-grid-layout/style.css

```diff
@@ -0,0 +1,102 @@
+:root 
+{
+    --main-padding: 5px;
+    --main-radius: 5px;
+}
+
+.container
+{
+    display: grid;
+    width: 100vw;
+    height: 100vh;
+
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 0.2fr 1.5fr 1.2fr 0.8fr;
+    grid-template-areas: 
+    "nav nav nav nav"
+    "sidebar main main main"
+    "sidebar content1 content2 content3"
+    "sidebar footer footer footer";
+    grid-gap: 0.2rem;
+
+    font-family: "Inter",sans-serif;
+    font-weight: 800;
+    font-size: 12px;
+
+    text-align: center;
+    text-transform: uppercase;
+    color: #004d40;
+}
+nav
+{
+    background: #987284;
+    grid-area: nav;
+
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+main
+{
+    background: #75b9be;
+    grid-area: main;
+
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+#sidebar
+{
+    background: #d0d6b5;
+    grid-area: sidebar;
+
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+#content1
+{
+    background: #f9b5ac;
+    grid-area: content1;
+
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+#content2
+{
+    background: #ee7674;
+    grid-area: content2;
+
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+#content3
+{
+    background: #e3170a;
+    grid-area: content3;
+    
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+footer
+{
+    background: #ffaf87;
+    grid-area: footer;
+    
+    padding-top: var(--main-padding);
+    border-radius: var(--main-radius);
+}
+
```

Comment:
you could add the following and remove the redundant lines in the above classes

nav, main, #sidebar, #content1, #content2, #content3, footer {
    padding-top: var(--main-padding);
    border-radius: var(--main-radius);
}

Reply:
done @AlishaPrasad ma'am